typed-racket-test/fail/issue-169-1.rkt

```diff
@@ -1,5 +1,5 @@
 #;
-(exn-pred #rx"expected: \\(Listof Nothing\\).*given: \\(Listof Pos\\*\\)")
+(exn-pred #rx"expected: \\Null.*given: \\(Listof Pos\\*\\)")
```

Comment:
`(Listof Nothing) ≡ Null` and now TR collapses accessible instances of `Nothing`